numbat/src/quantity.rs

```diff
@@ -347,7 +345,36 @@ impl PrettyPrint for Quantity {
     }
 }
 
+pub(crate) enum QuantityOrdering {
+    IncompatibleUnits,
+    NanOperand,
+    Less,
+    Equal,
+    Greater,
+}
```

Comment:
Would this lead to slightly less code?
```suggestion
pub(crate) enum QuantityOrderingResult {
    IncompatibleUnits,
    NanOperand,
    Ok(std::cmp::Ordering),
}
```